.github/workflows/push_pr.yml

```diff
@@ -25,4 +25,4 @@ jobs:
   compile_and_test:
     uses: apache/flink-connector-shared-utils/.github/workflows/ci.yml@ci_utils
     with:
-      flink_version: 1.16.0
+      flink_version: 1.19-SNAPSHOT
```

Comment:
Is the externalization based on the code from the `release-1.18` branch or master?

Reply:
initial commit is for master however let's see if it could be done for 1.18.0
there are several things which are required to be workaround like FLINK-32620, FLINK-25593 and some others

Reply:
yes, with last 2 commits 1.18 is supported.

there could be 2 ways: support it for 1.18.0 first only and then apply similar approach to adapt updates from 1.19
or support it for both 1.19 and 1.18  like here